sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py

```diff
@@ -127,7 +127,7 @@ async def list_registry_artifacts(self, **kwargs) -> AsyncItemPaged[RegistryArti
         last = kwargs.pop("last", None)
         n = kwargs.pop("page_size", None)
         orderby = kwargs.pop("order_by", None)
-        return await self._client.container_registry_repository.get_manifests(
+        return self._client.container_registry_repository.get_manifests(
```

Comment:
why is the `await` gone

Reply:
The list methods don't return a coroutine, they return an AsyncItemPaged

Reply:
oh yeah lmao, ignore me

Reply:
We can remove the "async" before the def list_registry_artifacts too